pkg/src/src/main/scala/edu/berkeley/cs/amplab/sparkr/sqlUtils.scala

```diff
@@ -0,0 +1,11 @@
+package edu.berkeley.cs.amplab.sparkr
+
+import org.apache.spark.SparkContext
+import org.apache.spark.sql.{SQLContext}
+import org.apache.spark.sql.types.{StructType}
+
+object sqlUtils {
```

Comment:
Another minor nit - For most Scala class names in Spark we use capital first letter. Actually to be consistent with SparkSQL I'd say we can make this `SQLUtils` ?